stable/mission-control/templates/mission-control-statefulset.yaml

```diff
@@ -117,6 +117,8 @@ spec:
         - name: applicationyaml
           mountPath: "/tmp/etc/application.yaml"
           subPath: application.yaml
+        resources:
+{{ toYaml .Values.initContainers.resources | indent 10 }}
```

Comment:
Add this to the values.yaml please

Reply:
@danielezer its already there - https://github.com/jfrog/charts/blob/master/stable/mission-control/values.yaml#L30